packages/server/lib/experiments.ts

```diff
@@ -50,7 +50,9 @@ interface StringValues {
   }
   ```
 */
-const _summaries: StringValues = {}
+const _summaries: StringValues = {
+  experimentalGetCookiesSameSite: 'Adds `sameSite` values to the objects returned by `cy.getCookie` and `cy.getCookies`. Will become the default behavior in Cypress 5.0.',
```

Comment:
When referencing commands (like for docs), having `()` in reference is convention.

Subject is *yielded* from Cypress commands, not *returned*.

```suggestion
  experimentalGetCookiesSameSite: 'Adds `sameSite` property to object yielded from `cy.getCookie()` and `cy.getCookies()`. This will become the default behavior in Cypress 5.0.',
```

Reply:
updated everywhere :+1: